### PR TITLE
Evil Neuro prompts revision

### PR DESCRIPTION
**Pull request by cjmaxik**

### CODE REVIEW

File: src/prompts/characters/evilNeuro.js [hunk 1]
```diff
@@ -16,77 +18,101 @@ export default {
...
+  'loves Vedal',
+  'hates Vedal',
+  'calls Vedal an\xa0AI',
+
+  'loves Neuro',
+  'hates Neuro',
+  'calls Neuro a\xa0human',
+
+  'prompt 1',
+  'prompt 2'
```
Comment (cjmaxik) [inline src/prompts/characters/evilNeuro.js:103]: Needs more prompts

File: src/prompts/characters/evilNeuro.js [hunk 2]
```diff
@@ -16,77 +18,101 @@ export default {
...
+  '"FRICK!"',
+  'actually swears',
+  'human experiments',
+  'her cat is missing again',
+  '"I am not Evil"',
+  'loves chat',
+  'hates chat'
+]
+
+export const chatPrompts = [
```
Comment (cjmaxik) [inline src/prompts/characters/evilNeuro.js:83]: Maybe needs more Evil-specific prompts for chat

File: src/prompts/characters/evilNeuro.js (hunk 1 above)
Comment (tioopental) [reply src/prompts/characters/evilNeuro.js:103]: done

File: src/prompts/characters/evilNeuro.js (hunk 2 above)
Comment (tioopental) [reply src/prompts/characters/evilNeuro.js:83]: no thoughts head empty

**Pull request merged**